test/src/ui/js/test_Game.js

```diff
@@ -1136,19 +1136,33 @@ test("test_Game.pageAddSkillListFooter", function(assert) {
   });
 });
 
-test("test_Game.pageAddLogFooter", function(assert) {
+test("test_Game.pageAddSkillListFooter", function(assert) {
```

Comment:
This diff is weird looking, so i may be confused about what's going on here, but did you delete the original "test_Game.pageAddLogFooter" test and replace it with this new test, instead of adding the new test and retaining the old one?  If so, why?